cookbooks/child-workflows/child-workflows-megafood.md

```diff
@@ -0,0 +1,321 @@
+# Instafood: Cadence Child Workflow Cookbook
+
+![Instaclustr Managed Cadence](images/Instaclustr_Product_Managed_Cadence.png)
+
+## Introduction
+
+### Who is this cookbook for?
+
+This cookbook is for developers and engineers of all levels looking to understand how child workflows work in Cadence. The
+recipe in this book provides *"Hello World!"* type examples based on simple scenarios and use cases.
+
+### What you will learn
+
+How to setup a simple Cadence application which implements child workflows on Instaclustr's Managed Service Platform.
+
+### What you will need
+
+- An account on Instaclustr’s managed service platform (sign up for a free trial using the
+  following [signup link](https://console2.instaclustr.com/signup))
+- Basic Java 11 and Gradle installation
+- IntelliJ Community Edition, Visual Studio Code or any other IDE with Gradle support
+- Docker (optional: only needed to run Cadence command line client)
+
+### What is Cadence?
+
+A large number of use cases span beyond a single request-reply, require tracking of a complex state, respond to asynchronous events, and communicate to external unreliable dependencies. The usual approach to building such applications is a hodgepodge of stateless services, databases, cron jobs, and queuing systems. This negatively impacts developer productivity as most of the code is dedicated to plumbing, obscuring the actual business logic behind a myriad of low-level details.
+
+Cadence is an orchestration framework that helps developers write fault-tolerant, long-running applications, also known as workflows. In essence, it provides a durable virtual memory that is not linked to a specific process, and preserves the full application state, including function stacks, with local variables across all sorts of host and software failures. This allows you to write code using the full power of a programming language while Cadence takes care of durability, availability, and scalability of the application.
```

Comment:
Maybe worth rephrasing this (and updating the other cookbook too). Cadence doesn't really "preserve" memory or function stacks -- rather it replays state. Which can achieve the same thing if everything is idempotent. But it's not like it hits save to disk on some VM thing and later hits resume.